frontend/scripts/react-components/tool-selector/tool-selector.scss

```diff
@@ -0,0 +1,20 @@
+@import 'styles/settings';
+
+.c-tool-selector {
+  margin-top: $nav-height;
+
+  .grid-list {
+    margin: 30px auto 0;
+    max-width: 780px;
+    flex-wrap: wrap;
+    display: flex;
+
+    .c-grid-list-item {
```

Comment:
this seems off, why not use the same size everywhere?

Reply:
I don't get this. This was the width on the designs

Reply:
You're right about that, but for uniformity's sake let's keep just one size. Pablo just gave me the green light 👍